gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -72,10 +72,15 @@ fluid.defaults("gpii.flowManager", {
 // Mixin grades for the FlowManager
 
 fluid.defaults("gpii.flowManager.local", {
+    // The grade to handle user login requests. Distributed from the GPII config files.
+    // In all-in-local config, the handler is "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler".
+    // In untrusted config, the handler should be "gpii.lifecycleManager.untrusted.stateChangeHandler".
+    logonStateChangeHandler: "gpii.lifecycleManager.userLogonHandling.matchMakingStateChangeHandler",
```

Comment:
Best to just define this in the relevant components themselves rather than make an extra place people need to look in order to find out what grades something has. And turn all the definitions of this into options distributions.

Reply:
This string still remains